crates/fluvio-cli/src/smartmodule/create.rs

```diff
@@ -34,8 +35,12 @@ impl CreateSmartModuleOpt {
             wasm: SmartModuleWasm::from_binary_payload(buffer),
             ..Default::default()
         };
+        
+        debug!("creating smart-module: {}", self.name);
         let admin = fluvio.admin().await;
         admin.create(self.name.to_string(), false, spec).await?;
+        println!("smart-module \"{}\" has been created.", self.name);
```

Comment:
why blackslash?

Reply:
That is how the message is structured in topic/create

`println!("topic \"{}\" created", name);`

Reply:
k